### DISCUSSION TITLE
how can I write metadata for my near nfts?

### DISCUSSION THREAD

**Discussion by algopsycho** [TENK-DAO/tenk#36] [Q&A] [answered]
[[](url)](https://nomicon.io/Standards/NonFungibleToken/Metadata) I need my meta to satisfy these conditions. does hash lips do this? thank you🤍

**Comment by starpause** [comment, marked as answer]
Hashlips in ETH mode is very close to being compliant with NEAR NFT standards (NEP-171). See full answer in https://github.com/TENK-DAO/tenk/discussions/30 thread.
